php74/Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM outrigger/apache-php-base
+
+RUN yum -y install \
+      https://rpms.remirepo.net/enterprise/remi-release-7.rpm && \
+    yum -y install \
+      gcc-c++ \
+      make \
+      php74 \
+      php74-php-bcmath \
+      php74-php-devel \
+      php74-php-fpm \
+      php74-php-gd \
+      php74-php-mbstring \
+      php74-php-mcrypt \
+      php74-php-pecl-memcached \
```

Comment:
In this PR https://github.com/phase2/docker-build/pull/47/files#diff-ce24a4205ce621c14ac1849960b386aaR38 you're bring in `php74-php-pecl-memcache`

Reply:
I copied the php73 files and then search/replaced them so I'm not sure why that difference is there. It seems like we would want to be consistent between them so I'll troll through the git logs and see if I can spot anything about why either one was picked. Good catch.